pt-BR/mail/chrome/messenger/am-server-top.dtd

```diff
@@ -2,12 +2,12 @@
 
 <!ENTITY serverSettings.label "Receber mensagens">
 <!-- LOCALIZATION NOTE (popServer.label) : do not translate "POP" in below line -->
-<!ENTITY popServer.label "POP Mail Server">
+<!ENTITY popServer.label "Servidor de emails POP">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY popServer.label "Servidor de mensagens POP">